priv/repo/migrations/20160805203929_create_skill.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CodeCorps.Repo.Migrations.CreateSkill do
+  use Ecto.Migration
+
+  def change do
+    create table(:skills) do
+      add :title, :string
+      add :description, :string
+      add :original_row, :integer
+      add :slug, :string
+
+      timestamps()
+    end
+
+    create index(:skills, [:slug], name: :index_skills_on_slug, unique: true)
```

Comment:
I still believe the correct thing to do is ensure this at the database level for consistency. What if we remove Phoenix? Remove `Inflex`? Remove `parameterize`?

I wouldn't rely on application-level logic for your data when your RDBMS can provide that additional sanity check.